data/homepage.yaml

```diff
@@ -5,6 +5,8 @@ banner:
   latest_version: Ansible 7 is now available.
   how_to_install: Install by copying the pip command.
 automate:
+  section_anchor: automate
```

Comment:
I think we'll need to put these in their own dict if we want to iterate on it, but that seems do-able, I can work on it.

Reply:
Cool. That sounds fine to me. I was thinking maybe creating a new dict but then we might need to move the `title` entry or duplicate it so I wasn't convinced.

Reply:
You're not wrong, but instead you have duplication on
```
blog:
  section_title: blog
```

So you can't win really. I think it's probably easier to hardcode the section IDs, if we change those we have a lot of work to do anyway - the YAML is meant for making text/image changes easy, not the layouts/HTML.